helmfile/charts/networkpolicy-service/templates/node-local-dns/allow-node-local-dns.yaml

```diff
@@ -0,0 +1,41 @@
+{{ if .Values.nodeLocalDns.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-node-local-dns
+spec:
+  podSelector:
+    matchLabels:
+      k8s-app: node-local-dns
+  policyTypes:
+    - Egress
+    - Ingress
+  ingress:
+    - from:
+        - namespaceSelector: {}
+      ports:
+        - port: 53
+          protocol: UDP
+        - port: 53
+          protocol: TCP
+    - from:
+        - podSelector:
+            matchLabels:
+              app: prometheus
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: monitoring
+      ports:
+        - port: 9253
```

Comment:
Do we gather metrics from node-local dns today?

Reply:
yes
we have a service for it too